app/models/user.rb

```diff
@@ -107,7 +107,7 @@ def validate
   def current_group_by_description=(group_description)
     if group_description
       desired_group = miq_groups.detect { |g| g.description == group_description }
-      desired_group ||= MiqGroup.find_by(:description => group_description) if super_admin_user?
+      desired_group ||= MiqGroup.in_region(region_id).find_by(:description => group_description) if super_admin_user?
```

Comment:
Is there a reason why you used in_region here instead of in_my_region?

Reply:
[What GM said](https://github.com/ManageIQ/manageiq/pull/16756#issuecomment-355676624)...